unimod_mapper/unimod_mapper.py

```diff
@@ -214,8 +214,14 @@ def name2first_mass(self, unimod_name):
         Returns:
             float: Unimod mono isotopic mass
         """
-        index = min(self.mapper.get(unimod_name, None))
-        return self._data_list_2_value(index, "mono_mass")
+        index = self.mapper.get(unimod_name, None)
+        if index is not None:
+            index = min(self.mapper.get(unimod_name, None))
```

Comment:
To avoid the repetition of `self.mapper.get(unimod_name, None)`, use `index_list `in line 217 and `index = min(index_list)` here

Reply:
Thanks for pointing it out :)